### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by sidvishnoi**
See https://github.com/karma-runner/karma-jasmine/issues/274

BREAKING CHANGE:
 Drop support for node 8 (add support for node 14)

### CODE REVIEW

**Comment by sidvishnoi** [comment]
@googlebot I signed it!

File: package.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
     "grunt-bump": "^0.8.0",
     "grunt-conventional-changelog": "^6.1.0",
     "grunt-conventional-github-releaser": "^1.0.0",
-    "grunt-eslint": "^23.0.0",
+    "grunt-eslint": "^22.0.0",
```
Comment (johnjbarton) [inline package.json:39]: Please:
  1) revert this line (back to 23.0.0)
  2) Update the engines to "node": ">= 10"
  3) Update .travis.yml to remove 8 and add 14
  4) Change the description to include 
```
BREAKING CHANGE:
  Drop support for node 8 (add support for node 14)
```

Comment (sidvishnoi) [reply package.json:39]: I don't mind, but do we surely want to make a breaking change to update a linting devDependency?

**Comment by johnjbarton** [comment]
Thanks!

**Pull request merged**